src/main/java/com/fulcrumgenomics/util/PickLongIndices.scala

```diff
@@ -63,12 +64,27 @@ private class IndexSet extends Iterable[Index] {
   def contains(index: String): Boolean = this.set.contains(index)
 }
 
-/** A Metric class for information about each picked index. */
+/**
+  * Output produced by `PickLongIndices` to describe the set of molecular indices picked by the program.
+  *
+  * @param index The molecular index sequence
+  * @param source The source of the sequence - either `existing` for indices provided to the program of
+  *               `novel` for indices generated by the program
+  * @param min_mismatches The smallest number of mismatches to any other reported molecular index.
+  * @param indices_at_min_mismatches The number of other reported molecular indices with `min_mismatches`
+  *                                  differences to this index.
+  * @param gc The fraction of the sequence composed of `G` and `C` bases.
+  * @param longest_homopolymer The length of the longest homopolymer within the index sequence.
+  * @param worst_structure_seq The sequence (including adapter plus index) that generated the worst (lowest energy)
+  *                            structure for this index.
+  * @param worst_structure_dbn The lowest energy structure in dot-bracket notation.
+  * @param worst_structure_delta_g The deltaG of the lowest energy structure.
```

Comment:
It's just left blank in the file if it's not computed.  I think it's fine as is since the documentation to the program explains these are not generated if ViennaRNA is not provided.